go.mod

```diff
@@ -263,5 +260,5 @@ replace (
 	github.com/cosmos/ledger-cosmos-go => github.com/cosmos/ledger-cosmos-go v0.12.4
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 	github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
-	github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v1.35.0-tm-v0.34.29
+	github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v1.36.0-tm-v0.34.29
```

Comment:
[not blocking] but highly encouraged to also bump this in the other go.mod files in this repo

Reply:
[22c9e94](https://github.com/celestiaorg/celestia-app/pull/3452/commits/22c9e94ba67cac65196e4adbd3094136bb71cb2e)